metaflow/plugins/kubernetes/kubernetes_decorator.py

```diff
@@ -384,13 +401,34 @@ def _save_package_once(cls, flow_datastore, package):
             )[0]
 
     @staticmethod
-    def parse_node_selector(node_selector: list):
+    def parse_kube_list(items: Union[str, List[str]], requires_both: bool = True):
```

Comment:
Is the name `parse_kube_list` seem confusing? Would something like `parse_kube_keyvalue_list` be better? It also makes the function argument `requires_both` more intuitive.

WDYT?

Reply:
cool, I made this change.